generator/generator-opc-generator/generators/app/templates/src/_index.ts

```diff
@@ -0,0 +1,16 @@
+import { LitElement, html, property, customElement } from 'lit-element';
+import style  from './<%= componentName %>.scss';
+
+@customElement('<%= componentName %>')
+export class <%= componentClass %> extends LitElement {
+  @property() name = 'World';
+  static get styles() {
+    return [ style ];
+  }
+
+  render() {
+    return html`
+      <p>Hello, ${this.name}!</p>
```

Comment:
@hybridx can we remove the styling from the generator and display text as `<div><component name></div>`

Reply:
It would mean that there would be no styles just the basic structure which makes sense because that's what it is supposed to do. 

changed it to
```
@property() name = '<%= componentName %>';
 <h2>${this.name}!</h2>
```